cmd/config/subcommand/task_resource_attribute_file_config.go

```diff
@@ -0,0 +1,86 @@
+package subcommand
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	cmdUtil "github.com/flyteorg/flytectl/pkg/commandutils"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/flyteorg/flytestdlib/logger"
+
+	"sigs.k8s.io/yaml"
+)
+
+// TaskResourceAttrFileConfig shadow config for TaskResourceAttribute.
+// The shadow config is not using ProjectDomainAttribute/Workflowattribute directly inorder to simplify the inputs.
+// As the same structure is being used for both ProjectDomainAttribute/Workflowattribute
+type TaskResourceAttrFileConfig struct {
+	Project  string `json:"project,omitempty"`
+	Domain   string `json:"domain,omitempty"`
+	Workflow string `json:"workflow,omitempty"`
+	*admin.TaskResourceAttributes
```

Comment:
omit-empty to this, instead of project domain attribute right?

Reply:
Done